CMakeLists.txt

```diff
@@ -1033,15 +1035,29 @@ if (CYGWIN)
 endif()
 
 message(STATUS "INSTALL DIRS: bin=${CMAKE_INSTALL_BINDIR} lib=${CMAKE_INSTALL_LIBDIR} shlib=${INSTALL_SHARED_DIR} include=${CMAKE_INSTALL_INCLUDEDIR}")
-
-install(TARGETS ${INSTALL_TARGETS}
-		RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
-		ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+if (NEED_DESTINATION)
+	if (DEFINED CMAKE_INSTALL_LIBDIR AND DEFINED CMAKE_INSTALL_LIBDIR AND NOT INSTALL_SHARED_DIR STREQUAL "")
```

Comment:
The same condition is used twice:
`CMAKE_INSTALL_LIBDIR` AND `CMAKE_INSTALL_LIBDIR`.

Reply:
Uh, nice catch!

Reply:
Fixed

Reply:
Hm... Somehow I still see the old version using two `CMAKE_INSTALL_LIBDIR`.

Reply:
Should be fixed now.